back/spec/acceptance/project_custom_fields_spec.rb

```diff
@@ -37,40 +37,10 @@
     }
   end
 
-  describe 'in a continuous ideation project without custom fields' do
-    let(:project) { create(:continuous_project) }
-    let(:project_id) { project.id }
-
-    before do
-      project.update!(input_term: 'question')
-    end
-
-    get 'web_api/v1/projects/:project_id/custom_fields/json_forms_schema' do
-      example_request 'Get the jsonforms.io json schema and ui schema for the custom fields' do
-        assert_status 200
-        json_response = json_parse response_body
-        expect(json_response.dig(:data, :type)).to eq 'json_forms_schema'
-        json_attributes = json_response.dig(:data, :attributes)
-        expect(json_attributes[:json_schema_multiloc].keys).to match_array %i[en fr-FR nl-NL]
-        %i[en fr-FR nl-NL].each do |locale|
-          expect(json_attributes[:json_schema_multiloc][locale][:properties].keys).to match_array(
-            %i[title_multiloc body_multiloc topic_ids location_description idea_images_attributes idea_files_attributes]
-          )
-        end
-        ui_schema = json_attributes[:ui_schema_multiloc][:en]
-        expect(ui_schema.keys).to match_array %i[type options elements]
-        expect(ui_schema[:type]).to eq 'Categorization'
-        expect(ui_schema[:options]).to eq({ formId: 'idea-form', inputTerm: 'question' })
-        expect(ui_schema[:elements].size).to eq 3
-        expect(json_attributes[:ui_schema_multiloc].keys).to match_array %i[en fr-FR nl-NL]
-      end
-    end
-  end
-
-  describe 'in a continuous ideation project with custom fields' do
-    let(:project) { create(:continuous_project) }
+  describe 'in a timeline project with an active ideation phase with custom fields' do
+    let(:project) { create(:project_with_active_ideation_phase) }
     let(:project_id) { project.id }
-    let!(:custom_form) { create(:custom_form, :with_default_fields, participation_context: project) }
+    let(:custom_form) { create(:custom_form, :with_default_fields, participation_context: project) }
```

Comment:
I have a slight preference to keep `let!` because we want to ensure that the default fields are created. If line 44 would be removed, it might be a bit hard to figure out why things are no longer working as expected.